packages/trader/src/AppV2/Layout/Routes/routes.tsx

```diff
@@ -1,9 +1,9 @@
 import { lazy } from 'react';
 import { routes } from '@deriv/shared';
-import Trade from 'AppV2/Containers/Trade';
 import Positions from 'AppV2/Containers/Positions';
 import { TRouteConfig } from 'Types';
 import ContractDetails from 'AppV2/Containers/ContractDetails';
+import Trade from 'AppV2/Containers/Trade';
```

Comment:
Should we modify this config for desktop? As it won't `have trader-positions`

Reply:
Probably yes, some thing like?

```
const mobileTraderRoutes: TRouteConfigExtended[] = [
    {
        path: routes.trade,
        component: Trade,
        exact: true,
        default: false,
    },
    {
        path: routes.trader_positions,
        component: Positions,
        is_authenticated: true,
        default: false,
    },
    {
        path: routes.contract,
        component: ContractDetails,
        is_authenticated: true,
        default: false,
    },
    {
        // default route
        path: '/*',
        component: lazy(() => import('Modules/Page404')),
        default: false,
    },
];

const desktopTraderRoutes: TRouteConfigExtended[] = [
    {
        path: routes.trade,
        component: Trade,
        exact: true,
        default: false,
    },
    {
        path: routes.contract,
        component: ContractDetails,
        is_authenticated: true,
        default: false,
    },
    {
        // default route
        path: '/*',
        component: lazy(() => import('Modules/Page404')),
        default: false,
    },
];
```

I would suggest to look at it later when we have the final designs.